composer/trainer/trainer.py

```diff
@@ -491,6 +491,26 @@ class Trainer:
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``1,048,675``)
         load_progress_bar (bool, optional): Display the progress bar for downloading the checkpoint.
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``True``)
+        load_ignore_keys (List[str] | (Dict) -> None, optional): A list of paths for the ``state_dict`` of the checkpoint,
+            which, when provided, will be ignored from the state_dict before a checkpoint is loaded. Each path is a list
+            of strings specifying the keys to index into ``state_dict`` joined together with `/` as a seperator.
+            If a prefix is provided, all children are also ignored (as demonstrated in Example 2). See :mod:`composer.core.state`
+            for the structure of state_dict.
+
+            Example 1: `load_ignore_model_keys = ["state/model/layer1/weights", "state/model/layer1/bias"]` would ignore
```

Comment:
```suggestion
            Example 1: `load_ignore_keys = ["state/model/layer1/weights", "state/model/layer1/bias"]` would ignore
```

Reply:
Might be useful in the example to use exactly what the torch format is for model weights, from @ravi-mosaicml 's comment, seems like it should be "state/model/layer1.bias", but would be good to verify?

This would also make clear why we chose `/` instead of `.`

Reply:
(I'd need to look at `model.named_parameters()` but that is my intuition)

Reply:
Yep, that's correct, can update examples accordingly.